components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -85,59 +85,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
             props.onInstanceUpdate(workspaceInstance);
         }
         switch (workspaceInstance?.status.phase) {
-            // unknown indicates an issue within the system in that it cannot determine the actual phase of
-            // a workspace. This phase is usually accompanied by an error.
-            case "unknown":
-                break;
-
             // Preparing means that we haven't actually started the workspace instance just yet, but rather
             // are still preparing for launch. This means we're building the Docker image for the workspace.
             case "preparing":
-                getGitpodService().server.watchWorkspaceImageBuildLogs(workspace!.id);
```

Comment:
I don't understand this change. Why would you like to not subscribe to image build logs when watching the prebuild logs while an image build is in progress? 🤔

Reply:
So we still do, we're just collapsing the cases. The diff looks confusing as it has no additions but we effectively remove all the empty `break` cases and only leave the "preparing" and "stopping" cases that had some logic

Reply:
Ah, right, it's a fall through. 👍